rust/stackablectl/src/cmds/stack.rs

```diff
@@ -140,7 +147,34 @@ impl StackArgs {
         debug!("Handle stack args");
 
         let transfer_client = xfer::Client::new_with(cache);
-        let files = cli.get_stack_files().context(PathOrUrlParseSnafu)?;
+
+        let release_files = cli.get_release_files().context(PathOrUrlParseSnafu)?;
+        let release_list = release::ReleaseList::build(&release_files, &transfer_client)
+            .await
+            .context(BuildListSnafu)?;
+
+        let release_branch = match &self.release {
+            Some(release) => {
+                if release == "dev" {
+                    "main".to_string()
+                } else {
+                    format!("release-{release}")
+                }
+            }
+            None => {
+                let release = release_list
+                    .inner()
+                    .first()
```

Comment:
I think we can not just always take the first item from the list (there can also be user-provided releases).
We can also probably also not sort them by semver because of that.
But we can probably avoid all of that by using `latest`